### DISCUSSION TITLE
how to add  iconv   in  the  3rdparty?

### DISCUSSION THREAD

**Discussion by haoqoo** [axmolengine/axmol#2232] [Q&A] [answered]
how to add  iconv   in  the  3rdparty?  
 The old project requires iconv for GBK encoding, but axmol has removed iconv. Can iconv be added

**Comment by rh101** [comment, marked as answer]
You can add any library you want to your own project.  This has nothing to do with Axmol.

So, add the iconv library somewhere in your project folder, and link it to your application in the `CMakeLists.txt` file.  

For example, if it is named `libiconv`, then you link it like this:

`target_link_libraries(${APP_NAME} libiconv)`

**Reply by haoqoo** [reply]
ok,thanks
